src/coroutine/memoset/mod.rs

```diff
@@ -499,15 +547,18 @@ impl<F: LurkField, Q: Query<F>> Scope<Q, LogMemo<F>> {
                         // It shouldn't exist, when instead we have only the single NIVC circuit repeated multiple times.
                         let cs = &mut cs.namespace(|| format!("chunk-{i}"));
 
-                        let mut circuit: CoroutineCircuit<'_, F, LogMemoCircuit<F>, Q> =
-                            CoroutineCircuit::new(
-                                self,
-                                memoset_circuit.clone(),
-                                chunk.to_vec(),
-                                *index,
-                                s,
-                                rc,
-                            );
+                        // Not used here
+                        let next_query_index = 0;
```

Comment:
What's going on here?

Reply:
This is analogous to multiframe's next_pc. It is there so that we can properly implement SuperNova's synthesize

Reply:
Right, so why are we setting it to 0? And why is it described as not being used here?

Reply:
Shouldn't it be a function of `index`? (Most simply `index + 1` to start with?)

Reply:
It's 0 because this is not used in this function. This function concatenates all circuits, it's not using Nova at all

Reply:
Check out this [line](https://github.com/lurk-lab/lurk-rs/pull/1115/files#diff-5153fb79aa5cb33975f9bd31b1644093c237ac2a4eb19e826ff41c2460b1ee33R284)